android-record/pom.xml

```diff
@@ -0,0 +1,96 @@
+    <parent>
+        <groupId>com.fasterxml.jackson.module</groupId>
+        <artifactId>jackson-modules-base</artifactId>
+        <version>2.16.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jackson-module-android-record</artifactId>
+    <name>Jackson module: Android Record Support</name>
+    <packaging>bundle</packaging>
+
+    <description>Support deserialization into records on Android</description>
+    <url>https://github.com/FasterXML/jackson-modules-base</url>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <!-- Generate PackageVersion.java into this directory. -->
+        <packageVersion.dir>com/fasterxml/jackson/module/androidrecord</packageVersion.dir>
+        <packageVersion.package>com.fasterxml.jackson.module.androidrecord</packageVersion.package>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+        </dependency>
+      <dependency>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-annotations</artifactId>
+      </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+          <plugin>
+               <groupId>org.apache.maven.plugins</groupId>
+               <artifactId>maven-compiler-plugin</artifactId>
+               <inherited>true</inherited>
+               <configuration>
+                 <compilerArgs>
+                   <arg>-parameters</arg>
+                 </compilerArgs>
+                 <fork>true</fork>
+                 <useIncrementalCompilation>true</useIncrementalCompilation>
+               </configuration>
+           </plugin>
+
+            <plugin>
+                <!-- Inherited from oss-base. Generate PackageVersion.java.-->
+                <groupId>com.google.code.maven-replacer-plugin</groupId>
+                <artifactId>replacer</artifactId>
+                <executions>
+                    <execution>
+                        <id>process-packageVersion</id>
+                        <phase>generate-sources</phase>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+            </plugin>
+
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>animal-sniffer-maven-plugin</artifactId>
+              <version>${version.plugin.animal-sniffer}</version>
```

Comment:
I guess this must come from `jackson-core` dependency?

Reply:
It comes from the project's main pom file. See [discussion above](https://github.com/FasterXML/jackson-modules-base/pull/227#issuecomment-1783636041).

Reply:
Oh, I missed that one (main-level pom at the bottom of diffs). Thanks!